tasks/th/thaipop.py

```diff
@@ -0,0 +1,122 @@
+
+# coding: utf-8
+
+# In[ ]:
+
+'''
+Bigmetadata tasks
+tasks to download and create metadata
+'''
+
+from tasks.tags import SubsectionTags, SectionTags
+from tasks.meta import (GEOM_REF, current_session, GEOM_NAME, OBSColumn)
+from tasks.util import ColumnsTask, TableTask, Carto2TempTableTask
+from collections import OrderedDict
+
+class ImportThai(Carto2TempTableTask):
+
+    subdomain = 'solutions'
+    table = 'thai_districts'
+
+
+# In[ ]:
+
+class ThaiColumns(ColumnsTask):
+
+    def requires(self):
+        return {
+            'sections': SectionTags(),
+            'subsections': SubsectionTags(),
+        }
+
+    def version(self):
+        return 1
+
+    def columns(self):
```

Comment:
It looks like CCA is a unique code identifier, but there are 924 unique codes whereas there are 928 names. Three of the four missing codes can be explained by three locations (two Songkhla Lakes and Wiang Kao) which have null for population and CCA. There might be a duplicate that would explain the last missing code.